docs: integrate supabase in dhiwise, steps (#9190)
supabase/supabase: apps/reference/docs/guides/integrations/dhiwise.mdx

---
id: dhiwise
title: 'DhiWise'

> description: 'Get started with Supabase and DhiWise, Covert your Figma design into Flutter apps, Authenticate Users with Supabase and Connect Supabase as a backend in your Flutter application using DhiWise'

Reviewer: ```suggestion
description: 'Get started with Supabase and DhiWise. Convert your Figma designs into Flutter apps, store data and authenticate your users.'
```

---
import select from "/img/guides/integrations/dhiwise/select.png"
import create from "/img/guides/integrations/dhiwise/create.png"

This guide explains how to quickly connect Supabase back-end to an DhiWise Flutter application.

> [DhiWise](https://www.dhiwise.com/) is a Developer tools to covert Figma designs into React and Flutter applications. It let's you integrate Database and APIs to your React and Flutter Apps quickly.

Reviewer: ```suggestion
[DhiWise](https://www.dhiwise.com/) is a Developer tool to convert Figma designs into React and Flutter applications. It lets you integrate Database and APIs into your React and Flutter Apps quickly.
```

> If you don’t have an DhiWise account, create one [here](https://app.dhiwise.com).

Reviewer: ```suggestion
If you don’t have a DhiWise account, create one [here](https://app.dhiwise.com).
```

DhiWise supports easy Supabase integration in just 5 steps.

Let’s get started!

## Step 1: SignIn to Supabase

Go to [Supabase](https://supabase.com/), Click on `Sign In` and and create a new account by authenticating with **GitHub**. If you already have an account, you will be logged in.

## Step 2: Create a new project in Supabase

From the Dashboard click on `New project` and select an organization. If you don’t have an organization then create one using `+ New organization`

- Give your Supabase project a `name`.
- Enter a secure `Database Password`.
- Choose the `region` where your app's backend is hosted.
- Click `Create new project`.

![New Project](/img/guides/integrations/dhiwise/newProj.png)

## step 3: Find the API key and URL

Once your project is created you can access the API Key and URL string, Or if you already have an account go to your `organization-> app-> settings-> API`.

![auth keys](/img/guides/integrations/dhiwise/authKeys.png)

## Step 4: Integrations

> Now, There are two ways you can integrate Supabase into DhiWise Flutter Application.

Reviewer: ```suggestion
There are two ways you can integrate Supabase into your DhiWise Flutter applications.
```
